boa3/model/builtin/interop/nativecontract/Gas/gasmethod.py

```diff
@@ -0,0 +1,14 @@
+import ast
+from typing import Dict, List
+
+from boa3.model.builtin.interop.nativecontract.Gas.getgasscripthashmethod import GasContract
+from boa3.model.builtin.interop.nativecontract.nativecontractmethod import NativeContractMethod
+from boa3.model.type.itype import IType
+from boa3.model.variable import Variable
+
+
+class GasMethod(NativeContractMethod):
```

Comment:
```suggestion
class GasContractMethod(NativeContractMethod):
```
just to make clear that these methods are from the GAS smart contract, not only related to the GAS token
it's going to be easier to differentiate other methods if included in this smart contract from those from NEP17 standard